### DISCUSSION TITLE
Is the new/big version is coming?

### DISCUSSION THREAD

**Discussion by pawlos** [NickeManarin/ScreenToGif#1230] [Q&A] [answered]
@NickeManarin I read in this [comment](https://github.com/NickeManarin/ScreenToGif/pull/1142#issuecomment-1436085318) that in the current v2 version only the bug fixing is considered. I was just wondering if the new version is still in the working?

**Comment by NickeManarin** [comment, marked as answer]
Yes, I have been working on it. :)
It's just taking some time to finish because I'm only working for a few hours on my weekends.

It's around 50%-ish done.
The editor is what's missing, along with some parts of Settings page.

![image](https://github.com/NickeManarin/ScreenToGif/assets/14798947/22c03088-b1ca-4842-a72f-4323c33ee474)

![image](https://github.com/NickeManarin/ScreenToGif/assets/14798947/c8e0535a-57a7-4190-b9a0-4aa2a18779ad)

**Reply by pawlos** [reply]
Thanks for the info. Looks good.

**Reply by pawlos** [reply]
@NickeManarin code for this new version is not yet available on github, right? will it be open source, or plan is to be something different?
